packages/dds/counter/src/test/fuzzUtils.ts

```diff
@@ -0,0 +1,108 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+
+import type {
+	AsyncGenerator as Generator,
+	Reducer,
+} from "@fluid-private/stochastic-test-utils";
+import {
+	combineReducers,
+	createWeightedAsyncGenerator as createWeightedGenerator,
+	takeAsync as take,
+} from "@fluid-private/stochastic-test-utils";
+import type { DDSFuzzModel, DDSFuzzTestState } from "@fluid-private/test-dds-utils";
+
+import type { IIncrementOperation } from "../counter.js";
+import { CounterFactory } from "../counterFactory.js";
+import type { ISharedCounter } from "../interfaces.js";
+
+/**
+ * Default options for Counter fuzz testing
+ */
+export const defaultOptions: Required<OperationGenerationConfig> = {
```

Comment:
```suggestion
export const defaultOptions: Required<DDSFuzzSuiteOptions> = {
```
and delete the custom type below